### PR TITLE
Fix ExternalReferencesDropdown layout

### PR DESCRIPTION
**Pull request by walterdeboer**
### Description

This PR moves the ExternalReferencesDropdown inside of the screen

### Addressed Issue

fixes: #539

### Checklist

- [x] I have read and understand the [contributing guidelines](https://github.com/DependencyTrack/dependency-track/blob/master/CONTRIBUTING.md#pull-requests)
- [ ] This PR introduces new or alters existing behavior, and I have updated the [documentation](https://github.com/DependencyTrack/dependency-track/tree/master/docs/_docs) accordingly

### CODE REVIEW

**Comment by nscuro** [comment]
Thanks @walterdeboer! 

Is there any way we can retain the original look and feel?

image

image

**Comment by walterdeboer** [comment]
Sure, you can style anything. But I wouldn't advice changing the CoreUI styles.. Next upgrade wil hurt more I guess

**Comment by walterdeboer** [comment]
P.S. i moved the > to the left on purpose. I didn't see any reason why it was on the right, now the dropdown menu is there...

**Comment by walterdeboer** [comment]
The style changed because I replaced the bootstrap-vue dropdown with the CoreUI dropdown.. I guess I could revert back to BootstrapVue with right alignment...

File: src/views/components/ExternalReferencesDropdown.vue [hunk 1]
```diff
@@ -1,20 +1,18 @@
...
-            <b-dropdown-item target="_blank" :href="r.url" v-b-tooltip.hover="{placement: 'left', title: r.url}">
-              <i class="fa fa-external-link text-muted"></i>&nbsp;
-              <span v-if="typeAcronyms.includes(r.type)" class="text-uppercase">{{ r.type }}</span>
-              <span v-else class="text-capitalize">{{ r.type }}</span>
-            </b-dropdown-item>
-          </span>
-        </ul>
-      </li>
-    </ol>
+    <b-dropdown size="sm" right>
```
Comment (nscuro) [inline src/views/components/ExternalReferencesDropdown.vue:3]: As a compromise between the previous styling and the default button style, how about this:

```suggestion
    <b-dropdown size="sm" class="mr-2" variant="outline-primary" right>
```

Renders to this:

image

**Review by nscuro — APPROVED** [review]

**Pull request merged**